lib/ansible/modules/remote_management/manageiq/manageiq_user.py

```diff
@@ -181,11 +181,17 @@ def compare_user(self, user, name, group_id, password, email):
         Returns:
             false if user fields have some difference from new fields, true o/w.
         """
+        # MIQ before 4.6 stored group membership in 'group' key
+        if user.get('group'):
+            user_group = user['group']
```

Comment:
it should be `user['group']['id']` ?

Reply:
yes it should be user['group']['id']. i've updated the commit accordingly